doc/ref_model/chapters/chapter01.md

```diff
@@ -95,7 +95,7 @@ This section introduces the high-level principles of infrastructure abstraction
    - Networking resources. (Limited to connectivity services only).
    - Acceleration resources.
 1. NFVI exposed resources should be supplier independent.
-1. All NFVI Application Programming Interfaces (API) must be standard and open to ensure component substitution if at all possible.
+1. All NFVI Application Programming Interfaces (API) must be standard and open to ensure component substitution if at all possible. An API is considered to be standard and open if it has an open source implementation what is widely accepted in production within the industry, with broad support by different vendors.
```

Comment:
Good clarification.  
Small grammar change request: "what is widely accepted" -> "that is widely accepted"

Reply:
Good start. 
> Given we're trying to be forward looking, do we want to preclude APIs which are not in production yet? I'm thinking we might want to loosen this up.
> Is it sufficient for the API itself to be published, or must it have an OSS implementation?

Reply:
Current 1.3 \ 4 text has four bullets. I wrote the 4th one with intention to replace previous ones by summarizing them and properly sorting in order of importance: desired outcome (What) is Interoperability, way to get there (How) is Simplification, Open Community (is one that has maintainers from competing companies and adoption downstream), and last is "standard" because SDOs often produce outcomes that don't ensure easy Interoperability.

Reply:
> 
> 
> Current 1.3 \ 4 text has four bullets. I wrote the 4th one with intention to replace previous ones by summarizing them and properly sorting in order of importance: desired outcome (What) is Interoperability, way to get there (How) is Simplification, Open Community (is one that has maintainers from competing companies and adoption downstream), and last is "standard" because SDOs often produce outcomes that don't ensure easy Interoperability.

Okay, I delete the first three bullets. The two big questions are:

1. Do we want to keep the standard development organisations? This would mean that we can have an API what have a standard definition but does not have an (open source) implementation.
1. De we want to remove the requirement that the implementation must be used in production?

Reply:
Do we really care how the "standard and open" APIs come about? 

"An API is considered to be standard and open if it is widely accepted within the industry, with broad support by both operators and vendors. Note, operators will, typically,  only deploy APIs within their environments if the APis are both standard and hardened."

Reply:
> 
> 
> Do we really care how the "standard and open" APIs come about?
> 
> "An API is considered to be standard and open if it is widely accepted within the industry, with broad support by both operators and vendors. Note, operators will, typically, only deploy APIs within their environments if the APis are both standard and hardened."

As we agreed to build on existing open source technologies I think we care how the API come about.

Reply:
"As we agreed to build on existing open source technologies ...", the definitions I have seen refer only to "open" but not on how they are built. Fir example, if a set of APIs are to be supported by some software component, will we be specifying that the component be built using open source technologies?

Reply:
> 
> 
> Current 1.3 \ 4 text has four bullets. I wrote the 4th one with intention to replace previous ones by summarizing them and properly sorting in order of importance: desired outcome (What) is Interoperability, way to get there (How) is Simplification, Open Community (is one that has maintainers from competing companies and adoption downstream), and last is "standard" because SDOs often produce outcomes that don't ensure easy Interoperability.

Hi Petar, Your new language is very well written and very concise, and I'm happy to approve it. Just to give some background, part of the reason for all of the redundancy in principle No. 4, which we've just optimized out, was to reinforce the mantra of _**open**_. Otherwise, language iterating through the various API uses would be located in the API chapter, as opposed to the principles chapter. :-)

Thank you for your contribution! -Mark

Reply:
> "As we agreed to build on existing open source technologies ...", the definitions I have seen refer only to "open" but not on how they are built. Fir example, if a set of APIs are to be supported by some software component, will we be specifying that the component be built using open source technologies?

My understanding is yes, this is why the API is called open. Any other opinions?

Reply:
I would like to see us ensure we are balanced in our definition of 'open' here.  One specific example I would cite for discussion is in a 5G study document, where they deliberately excluded the use of Protocol Buffers and gRPC as a communications mechanism, because it did not come from a "Standards Developing Organization".  (see [TR 29.890](https://www.3gpp.org/FTP/tsg_ct/WG3_interworking_ex-CN3/DRAFT_SPEC_IMPLEMENTATIONS/CT_79/29890-f10.doc) section 5.5.5.4 ).  The strong adherence to SDO requirement, here, excluded an implementation that was otherwise very well suited to the problem domain.

In reading the proposed text, it is unclear to me whether the clause "and come from Open Communities or Standards Development Organizations" would again exclude something like gRPC and Protocol Buffers.  Is it unclear to others? If so, then how would we resolve this?

Reply:
I tried to adjust the definition a bit based on your comments on chat.

Reply:
> ... deliberately excluded the use of Protocol Buffers and gRPC as a communications mechanism, because it did not come from a "Standards Developing Organization"...
Part "from Open Communities" is there so such examples can be included.

Reply:
> My understanding is yes, this is why the API is called open. Any other opinions?
There are various implementations of such APIs, some based on open source software which in most cases is not practical to check and other done with own implementation still allowing interoperability.

Reply:
> 
> 
> > My understanding is yes, this is why the API is called open. Any other opinions?
> 
> There are various implementations of such APIs, some based on open source software which in most cases is not practical to check and other done with own implementation still allowing interoperability.

Yes, I think the updated definition is aligned with this.